server/pulp/server/managers/content/orphan.py

```diff
@@ -161,26 +161,6 @@ def get_orphan(self, content_type_id, content_unit_id):
                                               content_unit=content_unit_id)
 
```

Comment:
@jortel, all the changes are just deletion of the code which was converted from pulp2 to pulp3 in this PR.
Two lines are "fixed" here to satisfy flake8.